xpf.go

```diff
@@ -0,0 +1,80 @@
+package dns
+
+import (
+	"fmt"
+	"net"
+)
+
+// XPF RR. See https://tools.ietf.org/html/draft-bellis-dnsop-xpf-04
+type XPF struct {
```

Comment:
Once you switch to adding a new field type to the generators, this should just go in `types.go`.